CMakeLists.txt

```diff
@@ -37,6 +37,30 @@ find_package(Vulkan REQUIRED)
 # Dependency setup with conan
 include(conan_setup)
 
+# The marked constants in application.hpp will be replaced with the following values
+set(ENGINE_NAME "Inexor Engine")
+set(APP_NAME "Inexor Vulkan-renderer example")
+
+set(ENGINE_VERSION_MAJOR 0)
+set(ENGINE_VERSION_MINOR 1)
+set(ENGINE_VERSION_PATCH 0)
+
+set(APP_VERSION_MAJOR 0)
+set(APP_VERSION_MINOR 1)
+set(APP_VERSION_PATCH 0)
```

Comment:
We should always prefix our variables with `INEXOR_`

Reply:
Done.